shopfloor_mobile/static/wms/src/scenario/delivery.js

```diff
@@ -12,6 +12,7 @@ const Delivery = {
     template: `
         <Screen :screen_info="screen_info">
             <template v-slot:header>
+                <state-display-info :info="current_location_msg()" v-if="current_location()"/>
```

Comment:
this msg should be set on state.display_info likely. Any reason to keep it separated?

Reply:
The message is set on display_info for the answer of a location scan (`scan_deliver` endpoint)

![image](https://user-images.githubusercontent.com/988765/180972567-fdc25888-e630-4ad5-ab7a-e59774046cd4.png)

But for all other calls it is not because it would require more changes on the back-end (each endpoint would need to receive the location id to send the information back).
But the front-end always knows on what sub-location it is working on by looking at the answer received by the last call to `scan_deliver`. And the sub-location is always displayed to the user.

![image](https://user-images.githubusercontent.com/988765/180973492-f117fcee-4bef-4f58-870a-83aecdbb5b46.png)

Reply:
> But for all other calls it is not because it would require more changes on the back-end (each endpoint would need to receive the location id to send the information back).

That's not true, there's a trick ;)
For zone picking I used an header for all the info that were mostly the same across the whole scenario.
Check these lines:

https://github.com/OCA/wms/blob/14.0/shopfloor/services/zone_picking.py#L95-L154